src/Cake.Issues.PullRequests.Tfs/TfsPullRequestSystemAliases.PullRequestSystem.cs

```diff
@@ -99,6 +99,69 @@ public static partial class TfsPullRequestSystemAliases
             return context.TfsPullRequests(new TfsPullRequestSystemSettings(repositoryUrl, pullRequestId, credentials));
         }
 
+        /// <summary>
+        /// Gets an object for writing issues to Team Foundation Server or Azure DevOps pull request
+        /// where all required data is taken from the appropriate system environment variable.
```

Comment:
I would mention here that it looks for the environment variables set by Azure DevOps / TFS build:

```
... is taken from the environment variables set by the Azure Pipelines / TFS build.
```

Reply:
Fixed.